CHANGELOG.md

```diff
@@ -53,4 +77,13 @@
 * **\[WIP\]** MbedTLS code.
 * **\[WIP\]** EMLib build-script for Kiso.
 
+ ## Known issues 
```

Comment:
```suggestion
### Known issues 
```
1) Maintain consistency of heading levels (level 3).
2) Leading space in front of `##` could cause render issues.